sites/example-project/src/components/ui/Sidebar.svelte

```diff
@@ -26,7 +26,7 @@
 <aside class="sidebar" class:open>
     <div class="sticky">
         <div class=nav-header>
-            <a href='/' on:click={() => open = !open}><h1 class=project-title>Evidence</h1></a>
+            <a href='/' on:click={() => open = !open}><div class=project-title>Evidence</div></a>
```

Comment:
Good idea. This is effectively the Logo / Home button.

I removed the H1 styling because with the change, H1 is now a lot larger and doesn't fit.

I'll mess around with H2 and see where I get